PROJECT

```diff
@@ -0,0 +1,13 @@
+domain: openshift.io
+layout: go.kubebuilder.io/v3
+projectName: assisted-service-operator
+repo: github.com/openshift/assisted-service-operator
```

Comment:
What is a PROJECT file? Why is it required? Does it have to have this name? Is this line correct?

Reply:
The PROJECT file is read by "operator-sdk generate bundle" to determine the project name. The value for this line isn't correct. After looking at this some more, we can remove this file and just pass in the project name when calling "operator-sdk generate bundle".

Reply:
I've corrected the repo url. Leaving the PROJECT file in the repo as it appears to be present in other operators' repos.